src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -27,6 +27,12 @@ public interface ITimeSyncBehaviorState
         /// <c>false</c> if the sample comes from a peer that our node connected to.</param>
         /// <returns><c>true</c> if the sample was added to the mix, <c>false</c> otherwise.</returns>
         bool AddTimeData(IPAddress peerAddress, TimeSpan offsetSample, bool isInboundConnection);
+
+        /// <summary>
+        /// Gets a value indicating whether the system time is not in sync and needs adjustment.
+        /// </summary>
+        /// <returns><true/> if the system time is not in sync and needs adjustment, <c>false</c> otherwise.</returns>
+        bool IsSystemTimeOutOfSync();
```

Comment:
But while the property with public get

Reply:
Fixed. I can't believe I created such a method.